### ISSUE TITLE
Question about Frescobaldi flatpak

### ISSUE THREAD

**Issue by papioara** [frescobaldi/frescobaldi#1772]
Hello everybody,

Today I have installed another flatpak update of Frescobaldi. On inspecting it, I found that at least the German translation in it is outdated with mixed German/English menus. There is, however, a complete localization in the Frescobaldi version which you get when you clone the repository. Since I do not know how flatpaks are being built and what is incorporated with which priority, I just want to draw your attention to this. And of course I am also saying that the flatpak should contain the most recent changes/fixes as well as compared to the repository version. Or are there reasons behind this which I am not aware of?

Thank you for a short comment...

**Comment by fedelibre** [comment]
The update of Frescobaldi flatpak contains only the latest LilyPond unstable version:

```
$ flatpak info org.frescobaldi.Frescobaldi | grep Subject
     Subject: lilypond-dev: Update lilypond.git to 2.25.18 (c2439f3e)
```

Nothing has changed in Frescobaldi itself. I can update Frescobaldi only when a new release is out.

**Comment by papioara** [comment]
Thank you for your explanation - now I understand better how it works behind the scene. Let's hope there will ever be a new version of Fexcobaldi...

**Comment by fedelibre** [comment]
Well, a solution to the current maintenance problem of Frescobaldi might be using the beta branch of Frescobaldi flatpak to get synced with Frescobaldi master. I once created the beta branch but never used it; this could be a good use case.

I'll look into this tonight and put you in Cc in the PR.

**Comment by fedelibre** [comment]
I've just merged [PR #56](https://github.com/flathub/org.frescobaldi.Frescobaldi/pull/56). It should be available in Flathub beta repository in a few hours.
